pages/services/kafka/2.6.0-2.2.1/security/index.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Securing your service
+title: Security
+menuWeight: 50
+model: /services/kafka/data.yml
+render: mustache
+enterprise: true
+---
+
+
+# DC/OS {{ model.techName }} Security
+
+-The DC/OS {{ model.techName }} service supports {{ model.techShortName }}'s native transport encryption, authentication, and authorization mechanisms. The service provides automation and orchestration to simplify the use of these important features.
+
+-A good overview of these features can be found [here](https://www.confluent.io/blog/apache-kafka-security-authorization-authentication-encryption/), and {{ model.techShortName }}'s security documentation can be found [here](http://kafka.apache.org/documentation/#security).
```

Comment:
Rewrite line #17: 
For more information on {{ model.techShortName }}'s security, read the following:
- [Security Overview](http://kafka.apache.org/documentation/#security)
- [Security Features](https://www.confluent.io/blog/apache-kafka-security-authorization-authentication-encryption/)

Kafka security, read [security features](https://www.confluent.io/blog/apache-kafka-security-authorization-authentication-encryption/) documentation.